### DISCUSSION TITLE
Autotypes not an available option for external db source

### DISCUSSION THREAD

**Discussion by gpalomar12** [Budibase/budibase#13130] [Help]
Hello,
       I've created several tables to be used for a CRM, I was looking into adding some autotype columns, but those don't seem to be available.  The data source I'm using is an AWS RDS Hosted Postgresql instance.  I fetched the data initially and wanted to add some created by, updated last, etc. columns.  These options don't appear to be available.  **Is this not available for external sources?**
       
       
     
![Autotypes_missing](https://github.com/Budibase/budibase/assets/62953814/7f1649e1-2a7b-48f6-9da0-dc88abdaca0c)

**Comment by ConorWebb96** [comment]
Hey @gpalomar12,

You can get creative here and use the [user](https://docs.budibase.com/docs/users-1) column type to mimic this. This column is an internal Budibase column.

All you would have to do is set your updated/created columns using the current user action and this would allow you to mimic the functionality shown in the internal tables auto columns.
<img width="322" alt="Screenshot 2024-02-26 at 11 53 47" src="https://github.com/Budibase/budibase/assets/126772285/5ceb5485-c24b-49cf-8240-7d79bec98511">

Hopefully this helps.
